meshtastic_flasher/form.py

```diff
@@ -305,7 +305,7 @@ def populate_languages(self):
         """
         self.select_language.addItem("Deutsch - German", "de")
         self.select_language.addItem("Español - Spanish", "es")
-        self.select_language.addItem("eestlane - Estonian", "et")
+        self.select_language.addItem("Eesti keel - Estonian", "et")
```

Comment:
Previously when we discussed this in Discord I did not have the full context, but in a list of languages I think it would be better to just have "Eesti"

Reply:
Great. No worries. I just changed it.